port.js

```diff
@@ -364,17 +366,24 @@ if (SAFARI) {
           for (var i = 0; i < result.locales.length; i++) {
             var locale = result.locales[i];
             var file = "_locales/" + locale + "/messages.json";
+            // Find translation messages for CatBlock
+            var cb_file = "catblock/_locales/" + locale + "/messages.json";
             // Doesn't call the callback if file doesn't exist
             syncFetch(file, function(text) {
               result.messages[locale] = JSON.parse(text);
             });
+            syncFetch(cb_file, function(text) {
+              var test = JSON.parse(text);
+              for (message in test) {
+                result.messages[locale][message] = test[message];
+              }
```

Comment:
What is this `test` variable? Please rename to make it clearer, and provide comments.